node-backend/app/routes/routes.js

```diff
@@ -6,11 +6,21 @@ const loginController = require("../controllers/loginController");
 require("../google_auth/google_auth")();
 const router = express.Router();
 router.use(bodyParser.urlencoded({ extended: true }));
-router
-  .route("/login")
-  .post(
-    passport.authenticate("google-token", { session: false }),
-    loginController.login
-  );
+router.post(
+  "/oauth/google",
+  (req, res, next) => {
+    if (req.body.access_token) {
```

Comment:
check if request body has access_token and it's value is not undefined or blank

Reply:
Where is checking of whether req.body has access token? And i will suggest to use yup here so it will handle all these things

Reply:
if else was actually checking for access_token. Now I will try it with yup